src/APIClient/GPURequestor.cpp

```diff
@@ -0,0 +1,25 @@
+#include "APIClient/GPURequestor.hpp"
+
+int APIClient::GPURequestor::putVulkanInfo(nlohmann::json vulkanInfosJson) {
```

Comment:
Why are you returning an `int`? It's not the same type as the prototype in the header file.